### ISSUE TITLE
Request for 'snapshot' beta versons for testing

### ISSUE THREAD

**Issue by herrold** [mgueury/html_validator#21]
I see in issue #9 that a 9.8.x exists, but do not see it at the web presence at
     http://users.skynet.be/mgueury/mozilla/download_090.html

Is there a way your workflow could be structured to do the build instructions at
   http://users.skynet.be/mgueury/mozilla/compile.html
and to place a installable testing candidate at GH ?

Thank you

**Comment by mgueury** [comment]
Thanks, sure. I have to redo the whole website. 

For now, it is here 
Firefox: https://addons.mozilla.org/en-US/firefox/addon/html-validator/
Chrome: https://chrome.google.com/webstore/detail/html-validator/mpbelhhnfhfjnaehkcnnaknldmnocglk
( Chrome version is not really unofficial, but working better than the Firefox one. Mostly because of https://github.com/mgueury/html_validator/issues/11 )

**Comment by herrold** [comment]
Thank you

Sadly, I get the message:
> This add-on requires a newer version of Firefox (at least version 57.0a1). You are using Firefox 52.0.

because I am on the LTS version of sources packaged and patched by Red Hat

When I have run the Mozilla.org 'developer' version under Linux, it was so 'leaky' that it ate up all ram, all swap, and then crashed.  I use my office browser as a tool not a hobby toy, or a developer's test-bed, and so could not move to a later version

is the 'FF 56' a hard requirement, or something that your code can accept an earlier version in future releases?

**Comment by mgueury** [comment]
I have made a fast update of the website.
Please check this:  http://users.skynet.be/mgueury/mozilla/compile.html
This will probably work for you.

**Comment by herrold** [comment]
Your instruction is most helpful.  Thank You

As I noted, I am version locked 'too early''  Is  'FF 56' a hard requirement

> There was an error during installation: Add-on {3b56bcc7-54e5-44a2-9b44-66c3ef58c13e} is not compatible with application version. add-on minVersion: 57.0a1.

I see this

> [herrold@centos-7 html_validator]$ egrep -r "57.0a1" *
manifest.json:      "strict_min_version": "57.0a1"

**Comment by mgueury** [comment]
I see that I forgot to answer this question.

I do not know the answer to your question. I have only tried with FF57 since it is in alpha/beta channel. It is quite probably that it works with FF56. But I have never tried.

I do not really want to update to the WebExtension for FF56 since the old version was more powerful. With the new Webextension API, there is a lack of API to do all what I did before.
 See the comparison table here:

http://users.skynet.be/mgueury/mozilla/roadmap.html
